src/assets/JSAsset.js

```diff
@@ -19,14 +19,15 @@ const SW_RE = /\bnavigator\s*\.\s*serviceWorker\s*\.\s*register\s*\(/;
 const WORKER_RE = /\bnew\s*Worker\s*\(/;
 
 class JSAsset extends Asset {
-  constructor(name, pkg, options) {
+  constructor(name, pkg, options, isProdBuild) {
     super(name, pkg, options);
     this.type = 'js';
     this.globals = new Map();
     this.isAstDirty = false;
     this.isES6Module = false;
     this.outputCode = null;
     this.cacheData.env = {};
+    this.isProdBuild = isProdBuild;
```

Comment:
This should normally be part of the this.options

Reply:
This is just the asset options, not the application options afaik

Reply:
Oh, you're right